src/prefect/utilities/compat.py

```diff
@@ -5,8 +5,6 @@
 
 import asyncio
 import sys
-from shutil import copytree
-from signal import raise_signal
```

Comment:
Hmm, it looks like removing these compat imports may be breaking the test suite and some other utilities